lib/templates/exportFile.js

```diff
@@ -117,6 +114,10 @@ class ExportFile {
       templateConfig.name_en = templateConfig.name;
       templateConfig.name_fr = templateConfig.name;
       delete templateConfig.name;
+
+      // Write updated config back to file
+      consola.debug(`Template ${templateConfig.name_nl}: adjusting name fields in config.json`);
+      fsUtils.writeConfig(this.TEMPLATE_TYPE, templateConfig.name_nl, templateConfig);
```

Comment:
I moved `writeConfig` to avoid calling it every time we call `read` (for example, every time you use an `update-` command). So in this case, we only write it when we are updating the names in it.
See the screenshot, the first time I run the command, we update names and write. The second time there is no need to update or write

![image](https://github.com/user-attachments/assets/e1e9a7e3-e3fa-4726-ad78-39f54c53aae0)